src/collections/_documentation/clients/go/usage.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Usage
+sidebar_order: 2
+---
+
+<!-- WIZARD -->
+## Capturing Errors
+
+In Go, there are both errors and panics, and Raven can handle both. To learn more about the differences, please read [Error handling and Go](https://blog.golang.org/error-handling-and-go).
+
+To handle normal `error` responses, we have two options: `CaptureErrorAndWait` and `CaptureError`. The former is a blocking call, for a case where you’d like to exit the application after reporting, and the latter is non-blocking.
+
+```go
+f, err := os.Open("filename.ext")
+if err != nil {
+    raven.CaptureErrorAndWait(err, nil)
+    log.Panic(err)
+}
+```
+
+## Capturing Panics
+
+Capturing a panic is pretty simple as well. We just need to wrap our code in `CapturePanic`. `CapturePanic` will execute the `func` and if a panic happened, we will record it, and gracefully continue.
+
+```go
+raven.CapturePanic(func() {
+    // do all of the scary things here
+}, nil)
+```
+<!-- ENDWIZARD -->
+
+Other than regular Errors and Panics, there are also two additional methods that allow sending information to Sentry.
+
+## Capturing Messages
+
+`CaptureMessage` and it's sibling `CaptureMessageAndWait` let you send arbitrary messages that will be logged in your Sentry's project.
```

Comment:
Fixed passive voice to active voice. Will you double check that I've used the correct subject? I assume the SDK is what does the logging. Here's the edit, "CaptureMessage and its sibling CaptureMessageAndWait let you send arbitrary messages that the SDK will log in your Sentry project."